src/Network/Network/help/Get-AzNetworkManagerVerifierWorkspaceReachabilityAnalysisRun.md

```diff
@@ -0,0 +1,293 @@
+                          "CreatedBy": "test@email.com",
+                          "CreatedByType": "User",
+                          "CreatedAt": "2024-09-19T16:07:08.2595595Z",
+                          "LastModifiedBy": "test@email.com",
+                          "LastModifiedByType": "User",
+                          "LastModifiedAt": "2024-09-19T16:07:18.9324364Z"
+                        }
+Id                    : /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/testRG/providers/Microsoft.Network/networkManagers/testNM/verifierW
+                        orkspaces/testVNV/reachabilityAnalysisRuns/testrun2
+
+Name                  : testrun3
+VerifierWorkspaceName :
+ResourceGroupName     : testRG
+NetworkManagerName    : testNM
+Properties            : Microsoft.Azure.Commands.Network.Models.NetworkManager.PSReachabilityAnalysisRunProperties
+Type                  : Microsoft.Network/networkManagers/verifierWorkspaces/reachabilityAnalysisRuns
+SystemData            : Microsoft.Azure.Commands.Network.Models.NetworkManager.PSSystemData
+SystemDataText        : {
+                          "CreatedBy": "test@email.com",
+                          "CreatedByType": "User",
+                          "CreatedAt": "2024-09-19T16:08:38.3602762Z",
+                          "LastModifiedBy": "test@email.com",
+                          "LastModifiedByType": "User",
+                          "LastModifiedAt": "2024-09-19T16:08:50.960041Z"
+                        }
+Id                    : /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/testRG/providers/Microsoft.Network/networkManagers/testNM/verifierW
+                        orkspaces/testVNV/reachabilityAnalysisRuns/testrun3
+
+Name                  : testrunfan1
+VerifierWorkspaceName :
+ResourceGroupName     : testRG
+NetworkManagerName    : testNM
+Properties            : Microsoft.Azure.Commands.Network.Models.NetworkManager.PSReachabilityAnalysisRunProperties
+Type                  : Microsoft.Network/networkManagers/verifierWorkspaces/reachabilityAnalysisRuns
+SystemData            : Microsoft.Azure.Commands.Network.Models.NetworkManager.PSSystemData
+SystemDataText        : {
+                          "CreatedBy": "test@email.com",
+                          "CreatedByType": "User",
+                          "CreatedAt": "2024-09-19T21:03:46.2977531Z",
+                          "LastModifiedBy": "test@email.com",
+                          "LastModifiedByType": "User",
+                          "LastModifiedAt": "2024-09-19T21:03:59.9659393Z"
+                        }
+Id                    : /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/testRG/providers/Microsoft.Network/networkManagers/testNM/verifierW
+                        orkspaces/testVNV/reachabilityAnalysisRuns/testrunfan1
+```
+Gets list of network manager verifier workspace reachability analysis runs for 'testVNV' verifier workspace.
+
+### Example 2
```

Comment:
in one of these examples, can you examine the properties and print out what an example of the results would look like? The backend returns them as a json; if there is any way we can format that json for users so its easier to read in their powershell window, that would be a great improvement